docs/en/observability/manage-synthetics-retention.asciidoc

```diff
@@ -0,0 +1,41 @@
+These indices are usually many times larger than the core metadata. Nhe relative sizes of each vary depending on the sites being 
+checked with network data usually being the larger of the two by a significant factor.
+
+`network` documents data consists of detailed metadata around requests for resources required by the pages being checked. An example would be an image, 
+a javascript file, or a font referenced and loaded by the page under test. 
+For each of these requests the URL, timing, headers, and other metadata are stored. While individually this metadata is small, 
+modern websites often request hundreds of additional resources per page load, adding up to a significant amount of storage.
+
+Screenshot data consists of binary image data used to construct a screenshot and metadata with information related to de-duplicating this data. De-duplication 
+efficiency makes screenshot data less burdensome than network data, especially for sites that are mostly visually unchanged across test runs. 
+De-duplication works by splitting each captured screenshot into an 8x8 grid of chunks each stored as a separate document keyed by a hash of its pixels. 
+This means that across checks to a given site only changes to the visual representation of the site require significant additional storage. If a site has not changed
+visually across runs the only additional storage required is the tiny `screenshot_ref` document for that image, which points to the relevant image blocks.
+
+[discrete]
+=== Managing the lifecycles of synthetics data streams
+
+Synthetics data is recorded in Elasticsearch [data streams](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html), an append-only
+structure in Elasticsearch. Synthetics data streams can be managed via the Elasticsearch API or via [Kibana index management](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-mgmt.html). 
+
+If Synthetics browser data streams are storing data longer than necessary, users can opt to retain `screenshot` and `network_info` datasets for a shorter period than the core metadata.
+To do so, first navigate to [Kibana index management](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-mgmt.html), then filter the list of data streams for
+those containing the term 'synthetics'. In the UI there will be three sorts of data stream present `synthetics-browser-*`, `synthetics-browser.network-*`, and `synthetics-browser.screenshot-*`. From this page you can retrieve the size of each data stream on disk, as well as which [index life cycle](https://www.elastic.co/guide/en/elasticsearch/reference/current/set-up-lifecycle-policy.html) is associated with it. 
+
+To change the retention period of a given data stream simply edit the life cycle policy associated with it after ensuring that policy does not apply to addition data streams whose
```

Comment:
> apply to addition data streams whose ...

@andrewvc  Should this be `additional` instead?

Reply:
Without screenshots, I think this is quite difficult to follow.  Also, it contains details on how to change the retention, but also nests info about how to create a new one (if it’s shared by something else where you don’t want to change the retention).

It may be easier to follow if these were split out?

Reply:
@colleenmcginnis @ollyhowell I intentionally omitted screenshots because they're hard to maintain... but I fear you're right. It _is_ very hard to follow without them. Does anyone on the docs team have thoughts there?

I'm inclined to just include them. Perhaps it's better to have them and let them get a little out of date if need be. Thoughts?

Reply:
It's ok to use screenshots when they are necessary! The fact that you've gone through this thought process before deciding they are necessary shows that you're not at risk of _overusing_ them. I think that's what the docs team is usually looking out for -- _unnecessary_ screenshots. 🙂